aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -160,6 +168,29 @@ module aptos_framework::jwks {
     // Functions begin.
     //
 
+    /// Called by a federated keyless dapp owner to install the JWKs for the federated OIDC provider (e.g., Auth0, AWS
+    /// Cognito, etc).
+    /// TODO: Should we be type-safe and explicitly use a `struct FederatedJWKs { jwks: AllProviderJWKs }` instead of
+    ///   reusing `PatchedJWKs { jwks: AllProviderJWKs }`, which is a JWK-consensus-specific struct? We'd
+    ///   need to be careful how we read it in Rust (but BCS serialization should be the same).
+    public fun install_federated_jwks(jwk_owner: &signer, patched_jwks: PatchedJWKs) acquires PatchedJWKs {
```

Comment:
i don't think we have helper functions for creating `PatchedJWKs`, so callers won't be able to get one?

Reply:
Oh my

Reply:
should it be an entry function so users only need to pass in bytes?

Reply:
I changed it to reuse our patching logic; see `jwks.move`